doc/ssl/SSL_CTX_set_tlsext_status_cb.pod

```diff
@@ -16,8 +21,10 @@ Certificate Status Request functions
  long SSL_CTX_set_tlsext_status_arg(SSL_CTX *ctx, void *arg);
 
  long SSL_CTX_set_tlsext_status_type(SSL_CTX *ctx, int type);
+ long SSL_CTX_get_tlsext_status_type(SSL_CTX *ctx);
 
  long SSL_set_tlsext_status_type(SSL *s, int type);
+ long SSL_get_tlsext_status_type(SSL *s);
```

Comment:
All good questions. I am enthusiastic to see the SSL_ctrl purgatory you suggest, and that was on my mental TODO list for post 1.1.0. The only reason to use SSL_ctrl here was for consistency. This function:

`long SSL_set_tlsext_status_type(SSL *s, int type);`

exists in the above form as an SSL_ctrl in 1.0.2. It therefore made logical sense to implement SSL_CTX_set_tlsext_status_type() in the same way for 1.1.0. With the setters implemented via SSL_ctrl, it then made logical sense for the getters to also be implemented in that way.

If we change it to not use SSL_ctrl() in 1.1.0 what exactly would we change? Would we keep the setters as SSL_ctrl, and the getters as stand alone functions? Or would we change everything introduced into 1.1.0 as a standalone function, but retain SSL_set_tlsext_status_type() as a macro because that was in 1.0.2? Or would we change everything including SSL_set_tlsext_status_type()?